CalibCalorimetry/HcalPlugins/python/Hcal_Conditions_forGlobalTag_cff.py

```diff
@@ -209,7 +208,7 @@
 run3_HB.toModify( es_hardcode, useHBUpgrade = cms.bool(True), HBreCalibCutoff = cms.double(100.0) )
 
 # now that we have an emap
```

Comment:
@kpedro88 not being involved too much I would think that this comment is not correct, since you apply `_toGet_noEmap` and not `_toGet`

Reply:
It means we have an emap from the GT, so we don't need to get it from the hardcode conditions.

Reply:
I see